packages/components/src/primitives/svg/index.native.js

```diff
@@ -1,13 +1,33 @@
+/**
+ * External dependencies
+ */
+import { omit } from 'lodash';
 import {
-	Path as PATH,
+	Path,
+	G,
 	Svg,
 } from 'react-native-svg';
 
-export const Path = PATH;
+export {
+	G,
+	Path,
+};
+
 export const SVG = ( props ) => {
+	// for some reason if a `className` prop is passed it gets converted to `style` here.
```

Comment:
citing the comment: `Given it carries a string (as it was originally className) but an object is expected for "style"`... it chokes :)

Reply:
Also thanks for pointing that out (about the PR where this change is implemented). We discussed that elsewhere and forgot to add this here. It might be good to change the comment to add this information in particular, adjusting the _"for some reason..." _ part.

Reply:
Comment modified in 522785990